windows.md

```diff
@@ -0,0 +1,29 @@
+Windows Node Support
+
+Windows nodes are partially supported by kubecost as of v1.93.0. Additional support is coming soon!
+* Deployment
+  * The cluster must have at least 1 linux node for the kubecost cost-model to run on
+  * Taint the cost-model deployment's pods to run on this node, eg:
+    ```
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      containers:
+    ```
+  * Taint the network-costs and node-exporter daemonset to run on linux nodes only, if using network-costs and node-exporter, eg:
+    ```
+      affinity: 
+        nodeAffinity:  
+          requiredDuringSchedulingIgnoredDuringExecution:  
+            nodeSelectorTerms:  
+            - matchExpressions:  
+              - key: kubernetes.io/os 
+                operator: In 
+                values: 
+                - linux
+    ```
+
+* Metrics
+  * When tracked by the kubernetes API, nodes should show up with the correct number of pods and resources.
+  * By default, we will be missing utilization data for pods on window nodes; pods will be billed based on request size.
```

Comment:
why's this? does cadvisor now collect windows metrics?

Reply:
should this just be combined with the point below?

Reply:
cadvisor can collect some windows metrics...it's just that support is a bit weak: https://github.com/google/cadvisor/issues/2158